lib/global_uid/base.rb

```diff
@@ -113,8 +113,9 @@ def self.notify(exception, message)
       end
     end
 
-    def self.get_connections
```

Comment:
Does this breaking change warrant a mention in the changelog?

Reply:
It will, the interface is changing significantly in this PR, I've got a few more commits coming and I'm planning to update the changelog at the end.